README.md

```diff
@@ -56,7 +56,9 @@ If you wish you can run this command under a virtual environment.
    instance).
 
  * Requirements resolver: Avocado has implemented the 'requirements resolver'
-   feature that makes easy to define test requirements. For now it is still in
+   feature that makes easy to define test requirements. Currently,
+   RequirementsResolver can handle `assets` and `packages` but new requirements
```

Comment:
> RequirementsResolver

the RequirementsResolver

> `assets`

can you give a brief example on what is meant by an 'asset'?

Reply:
Any file downloaded on internet, where avocado will make a cache. I will add this sentence there.